packages/botonic-cli/src/util/file-system.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  existsSync,
+  mkdirSync,
+  mkdtempSync,
+  readdirSync,
+  readFileSync,
+  rmdirSync,
+  writeFileSync,
+} from 'fs'
+import { copySync } from 'fs-extra'
+
+export function pathExists(path: string): boolean {
+  return existsSync(path)
+}
+
+export function readDir(path: string): string[] {
+  return readdirSync(path)
+}
+
+export function readJSON(path: string): any {
```

Comment:
the fact is that an object is not JSON if we are strict.
e.g.: 
```
{ dummy: 'content' }
```
is not:
```
{ "dummy": "content"}
```
so when using for example writeJSON
```
writeJSON(path, {
      dummy: 'content',
} as JSON)
``` 
won't work: `Conversion of type '{ dummy: string; }' to type 'JSON' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.`

Reply:
just read your comment for typing JSON 👍